scripts/environment_json.sh

```diff
@@ -41,74 +47,26 @@ while getopts rp:h? opt; do
 done
 shift $((OPTIND-1))
 
-function pull_from_config_xml() {
-  cat << EOM | node
-const fs = require('fs');
-const xml2js = require('xml2js');
-const s = fs.readFileSync('config.xml');
-xml2js.parseString(s, function(err, result) {
-  console.log($1);
-});
-EOM
-}
+if [[ "${TYPE}" == "release" && -z ${SENTRY_DSN:-} ]]; then
+  echo "SENTRY_DSN is undefined." 1>&2
+  exit 1
+fi
 
-function pull_from_plist() {
-  cat << EOM | node
-const fs = require('fs');
-const xml2js = require('xml2js');
-const s = fs.readFileSync('$1');
-xml2js.parseString(s, function(err, result) {
-  const dict = result['plist']['dict'][0];
-  const keys = dict['key'];
-  const values = dict['string'];
-  console.log(values[keys.indexOf('$2')]);
-});
-EOM
-}
+APP_VERSION="$(node "$(dirname "$0")/get_version.mjs" "${PLATFORM}")"
+APP_BUILD_NUMBER="$(node "$(dirname "$0")/get_build_number.mjs" "${PLATFORM}")"
```

Comment:
silly bash question, but this is local to this script unless i `export` it, correct?

Reply:
It changes the current process, so yes.
You use `export` to make it available to the subprocesses.